openshift/release/update-to-head.sh

```diff
@@ -1,19 +1,24 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-# Usage: update-to-head.sh release-0.5
+# Synchs the release-next branch to master and then triggers CI
+# Usage: update-to-head.sh
 
-target=$1
+command -v hub >/dev/null 2>&1 || { echo >&2 "I require hub (https://github.com/github/hub) but it's not installed.  Aborting."; exit 1; }
```

Comment:
Could we make that a less hard requirement ? :pray: (aka if `hub` is not present, add a text below to say "you gotta open a PR by visiting "that" or getting `hub` on your laptop")

Reply:
Hey @vdemeester, yes I can do that. The the script can be run manually as well